packages/stk/stk_ngp_test/stk_ngp_test/Reporter.hpp

```diff
@@ -43,28 +43,15 @@ inline std::ostream& operator<<(std::ostream& out, const Report& r) {
   return out;
 }
 
-class ReporterBase {
- public:
-  virtual ~ReporterBase() = default;
-  virtual void clear() = 0;
-  virtual void resize(int n) = 0;
-  virtual int get_capacity() const = 0;
-  virtual int report_failures(std::ostream& out = std::cout) = 0;
-
-  NGP_TEST_FUNCTION
-  virtual void add_failure(const char* condition,
-                           const char* location) const = 0;
-};
```

Comment:
@alanw0, are there places where this abstract base class for `Reporter` is needed? I didn't find anything extending this outside of STK tests.

Reply:
No, I can't find any other usages of ReporterBase. So if tests pass, I guess it is safe to remove.